Source/vtkPyFR/vtkPyFRPipeline.cxx

```diff
@@ -645,31 +735,35 @@ int vtkPyFRPipeline::CoProcess(vtkCPDataDescription* dataDescription)
     this->InsituLink->InsituUpdate(dataDescription->GetTime(),
                                    dataDescription->GetTimeStep());
 
-    vtkUpdateFilter(this->Contour, dataDescription->GetTime());
-    vtkUpdateFilter(this->Slice, dataDescription->GetTime());
+      vtkUpdateFilter(this->Contour, dataDescription->GetTime());
+      vtkUpdateFilter(this->Slice, dataDescription->GetTime());
+
     if(this->PyData(dataDescription)->PrintMetadata()) {
-        double* bds = this->ContourMapper->GetBounds();
+        double* bds = this->ActiveMapper1->GetBounds();
```

Comment:
I also like going back to SliceMapper.
But to Arvind's later question: this is indeed only for the message.  Min/Maxing both bounds would be pedantic.  In your case, though, I think you know the simulation so well that you'll never need the metadata, so this is a moot point.  I recommend you set metadata_out to False in your .ini and then never worry about this stuff.